docs/user-guide/area_calc.ipynb

```diff
@@ -6,9 +6,6 @@
    "source": [
```

Comment:
<p><code>uxarray/get_quadratureDG.py</code></p><p> </p><p>This file no longer exists after our refactors. Can you reference the updated one?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>